content/client-lib-development-guide/features.textile

```diff
@@ -270,7 +273,9 @@ h3(#realtimeclient). RealtimeClient
 * @(RTC6)@ @RealtimeClient#time@ function:
 ** @(RTC6a)@ Proxy to @RestClient#time@ presented with an async or threaded interface as appropriate
 * @(RTC7)@ The client library must use the configured timeouts specified in the @ClientOptions@, falling back to the "client library defaults":#defaults and defaults described in @ClientOptions@ below
-* @(RTC8)@ Authentication token changes for the current connection are possible when the client is in the @CONNECTED@, @CONNECTING@ or @DISCONNECTED@ states. If @auth.authorise@ is called with @AuthOptions#force@ set to @true@ and the state is @CONNECTED@, @CONNECTING@, or @DISCONNECTED@, the client will obtain a new token, disconnect the current transport and resume the connection (see "@RTN15c@":#RTN15c for details on resume). A test should exist that performs an upgrade of capabilities without any message loss during the upgrade process. A test should also exist for a token change that fails due to an incompatible token, which should result in the connection entering the @FAILED@ state
+* @(RTC8)@ Authentication token changes for the current connection are possible when the client is in the @CONNECTED@, @CONNECTING@ or @DISCONNECTED@ states. If @auth#authorize@ is called or Ably requests a re-authentication (see "RTN22":#RTN22), the client must obtain a new token and send an @AUTH@ @ProtocolMessage@ to Ably. The @data@ attribute of the @ProtocolMessage@ must contain either a token string or an encoded @TokenDetails@ object with the @encoding@ attribute set accordingly
```

Comment:
> can you be a bit more specific about what exactly you are suggesting I change?

For consistency with the existing connect-time params wouldn't it be just an `accessToken` member? Or if we want some structure to limit the proliferation of top-level members in the `ProtocolMessage` then an `auth` member with value `{ accessToken: xxx}`